python/ct/client/db_reporter.py

```diff
@@ -44,15 +46,15 @@ def _batch_scanned_callback(self, result):
 
 
 def _process_certs(db, log_key, certs_queue):
-    while True:
-        certs = certs_queue.get()
-
+    certs = certs_queue.get()
```

Comment:
Personally: I like Rob's approach to the loop better.
He's writing a "forever" loop, with a break, and that's indicated with while(True)
In your version you end up with a pre-loop 'get' call which is then repeated in the loop body so it can decide whether to continue or not.  I prefer the 'forever' construct over the repeated codel